react/features/remote-video-menu/components/web/MuteEveryoneElseButton.js

```diff
@@ -0,0 +1,77 @@
+/* @flow */
+
+import React from 'react';
+
+import { createToolbarEvent, sendAnalytics } from '../../../analytics';
+import { openDialog } from '../../../base/dialog';
+import { translate } from '../../../base/i18n';
+import { IconMicDisabled } from '../../../base/icons';
+import { connect } from '../../../base/redux';
+
+import AbstractMuteButton, {
+    _mapStateToProps,
+    type Props
+} from '../AbstractMuteButton';
+
+import RemoteVideoMenuButton from './RemoteVideoMenuButton';
+import MuteEveryoneDialog from '../../../remote-video-menu/components/web/MuteEveryoneDialog';
```

Comment:
Please arrange the imports according to https://github.com/jitsi/jitsi-meet/blob/master/doc/coding-style.md#formatting

`../AbstractMuteButton` and `./RemoteVideoMenuButton` are from the same feature, they need to be grouped last. And `'../../../remote-video-menu/components/web/MuteEveryoneDialog'` goes to the second group.

Reply:
Also why not import directly the feature?

Reply:
That is prone to intoducing cycles, importing the dialog directly is not a bad idea.

Reply:
For now we have this practice only if there isn't any other obvious solution. Do we have this problem here? If not we should definitely import the feature, at least for the sake of consistency.

Reply:
@gimre Could you please either change the import to use the feature or either add a comment why this is problematic?

Reply:
but wait actually this is the same exact directory, isn't it? Why not import `./MuteEveryoneDialog`?

Reply:
changed to import sibling './MuteEveryoneDialog'